packages/bot/src/transformers/interaction.ts

```diff
@@ -220,55 +223,62 @@ const baseInteraction: Partial<Interaction> & BaseInteraction = {
     interaction.authorizingIntegrationOwners = {}
 
-    if (payload.authorizing_integration_owners['0'])
+    if (payload.interaction.authorizing_integration_owners['0'])
       interaction.authorizingIntegrationOwners[DiscordApplicationIntegrationType.GuildInstall] = bot.transformers.snowflake(
-        payload.authorizing_integration_owners['0'],
+        payload.interaction.authorizing_integration_owners['0'],
       )
-    if (payload.authorizing_integration_owners['1'])
+    if (payload.interaction.authorizing_integration_owners['1'])
       interaction.authorizingIntegrationOwners[DiscordApplicationIntegrationType.UserInstall] = bot.transformers.snowflake(
-        payload.authorizing_integration_owners['1'],
+        payload.interaction.authorizing_integration_owners['1'],
       )
   }
-  if (props.context && payload.context) interaction.context = payload.context
-  if (props.data && payload.data) {
+  if (props.context && payload.interaction.context) interaction.context = payload.interaction.context
+  if (props.data && payload.interaction.data) {
     interaction.data = {
-      type: payload.data.type,
-      componentType: payload.data.component_type,
-      customId: payload.data.custom_id,
-      components: payload.data.components?.map((component) => bot.transformers.component(bot, component)),
-      values: payload.data.values,
-      id: payload.data.id ? bot.transformers.snowflake(payload.data.id) : undefined,
-      name: payload.data.name,
-      resolved: payload.data.resolved ? transformInteractionDataResolved(bot, payload.data.resolved, guildId) : undefined,
-      options: payload.data.options?.map((opt) => bot.transformers.interactionDataOptions(bot, opt)),
-      targetId: payload.data.target_id ? bot.transformers.snowflake(payload.data.target_id) : undefined,
-      // guildId: payload.data.guild_id ? bot.transformers.snowflake(payload.data.guild_id) : undefined,
+      type: payload.interaction.data.type,
+      componentType: payload.interaction.data.component_type,
+      customId: payload.interaction.data.custom_id,
+      components: payload.interaction.data.components?.map((component) => bot.transformers.component(bot, component)),
+      values: payload.interaction.data.values,
+      id: payload.interaction.data.id ? bot.transformers.snowflake(payload.interaction.data.id) : undefined,
+      name: payload.interaction.data.name,
+      resolved: payload.interaction.data.resolved ? transformInteractionDataResolved(bot, payload.interaction.data.resolved, guildId) : undefined,
+      options: payload.interaction.data.options?.map((opt) => bot.transformers.interactionDataOptions(bot, opt)),
+      targetId: payload.interaction.data.target_id ? bot.transformers.snowflake(payload.interaction.data.target_id) : undefined,
+      // guildId: payload.interaction.data.guild_id ? bot.transformers.snowflake(payload.interaction.data.guild_id) : undefined,
```

Comment:
```suggestion
```
Why is this commented?

Reply:
idk, it was there, it left it alone, the change is bc while doing CTRL + D in vsc it selected that too

Reply:
Unrelated: What's Ctrl+D?

Reply:
Shortcut for Find Match in vscode

If you select a word (like "payload.") and press CTRL + D will find the next "payload." and add a cursor on that, there is also a shortcut for all matches